build-it

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+mvn clean
+mvn clean install
+patch -p0 < ASTParser.patch
```

Comment:
I know this is a POC, but this really needs to be part of the maven build cycle through some plugin, or it should be a PR for javacc: https://github.com/javacc/javacc